signal2html/html_colors.py

```diff
@@ -75,6 +75,65 @@
     "group_color": GROUP,
 }
 
+# Extracted from:
+# https://github.com/signalapp/Signal-Android/blob/master/app/src/main/java/org/thoughtcrime/securesms/conversation/colors/AvatarColor.java
+# Note that Signal uses ColorInts, which are AARRGGBB. We strip off the alpha.
+AVATAR_COLORS = {
+    "C000": "#D00B0B",
+    "C010": "#C72A0A",
+    "C020": "#B34209",
+    "C030": "#9C5711",
+    "C040": "#866118",
+    "C050": "#76681E",
+    "C060": "#6C6C13",
+    "C070": "#5E6E0C",
+    "C080": "#507406",
+    "C090": "#3D7406",
+    "C100": "#2D7906",
+    "C110": "#1A7906",
+    "C120": "#067906",
+    "C130": "#067919",
+    "C140": "#06792D",
+    "C150": "#067940",
+    "C160": "#067953",
+    "C170": "#067462",
+    "C180": "#067474",
+    "C190": "#077288",
+    "C200": "#086DA0",
+    "C210": "#0A69C7",
+    "C220": "#0D59F2",
+    "C230": "#3454F4",
+    "C240": "#5151F6",
+    "C250": "#6447F5",
+    "C260": "#7A3DF5",
+    "C270": "#8F2AF4",
+    "C280": "#A20CED",
+    "C290": "#AF0BD0",
+    "C300": "#B80AB8",
+    "C310": "#C20AA3",
+    "C320": "#C70A88",
+    "C330": "#CB0B6B",
+    "C340": "#D00B4D",
+    "C350": "#D00B2C",
+    "crimson": "#CF163E",
+    "vermilion": "#C73F0A",
+    "burlap ": "#6F6A58",
+    "forest ": "#3B7845",
+    "wintergreen": "#1D8663",
+    "teal": "#077D92",
+    "blue": "#336BA3",
+    "indigo ": "#6058CA",
+    "violet ": "#9932CB",
+    "plum": "#AA377A",
+    "taupe": "#8F616A",
+    "steel": "#71717F",
+    "unknown": "#71717F",
+}
+
+
+def get_color(name):
+    return COLORMAP.get(name, None) or AVATAR_COLORS[name]
```

Comment:
Perhaps we can have a failsafe here:
```
color = COLORMAP.get(name, None) or AVATAR_COLORS.get(name, None)
if color is None:
    logger.warning(f"Unknown color {name}")
    return AVATAR_COLORS["unknown"]
else:
    return color
```

Reply:
Yeah that's a good idea. I had in mind that it would crash and we'd hear about it, but a warning is probably more user friendly :)

Reply:
Fixed in f57bbbf